CHANGELOG.md

```diff
@@ -11,6 +11,7 @@
 ## Changed
 
 ## Fixed
+- **decidim-proposals**: Fix for the proposals that have theirs state not published [#5832](https://github.com/decidim/decidim/pull/5832)
```

Comment:
We need a blank line between the title and the entry, otherwise codeclimate will complain.